packages/icons-workflow/bin/build.js

```diff
@@ -19,7 +19,7 @@ import prettier from 'prettier';
 import Case from 'case';
 import { fileURLToPath } from 'url';
 
-import systemsIconMapping from './icons-mapping.json' assert { type: 'json' };
+import systemsIconMapping from './icons-mapping.json' with { type: 'json' };
```

Comment:
This was throwing errors - assert was the previous / experimental syntax; it's since been updated to `with` which is the latest specc'ed use.

Reply:
I was able to pull this out - as long as the user isn't on a new version of node the old syntax works fine!